x-pack/test/api_integration/apis/entity_store/host.ts

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import {
+  ELASTIC_HTTP_VERSION_HEADER,
+} from '@kbn/core-http-common';
+import { FtrProviderContext } from '@kbn/ftr-common-functional-services';
+import type { GetEntityStoreStatusResponse } from '../../../../solutions/security/plugins/security_solution/common/api/entity_analytics/entity_store/status.gen'
+
+const DATASTREAM_NAME: string = 'logs-elastic_agent.cloudbeat-test';
+const HOST_TRANSFORM_ID: string = 'entities-v1-latest-security_host_default';
+const INDEX_NAME: string = '.entities.v1.latest.security_host_default';
+const TIMEOUT_MS: number = 300000; // 5 minutes
+
+export default function (providerContext: FtrProviderContext) {
+  const supertest = providerContext.getService('supertest');
+  const retry = providerContext.getService('retry');
+  const es = providerContext.getService('es');
+
+  describe('GET /api/entity_store/status', () => {
+
+    describe('not_installed', () => {
+      it("Should return 200 and status 'not_installed'", async () => {
+        const { body } = await supertest
+          .get('/api/entity_store/status')
+          .expect(200);
+
+        const response: GetEntityStoreStatusResponse = body as GetEntityStoreStatusResponse
```

Comment:
Comment: Not sure how to properly type known API responses and whether that helps.